.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,26 @@ assignees: ''
 
 ---
 
-<!-- Thanks for helping us to improve k3s! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->
+<!-- Thanks for helping us to improve K3s! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
 
-**Version:**
-<!-- Provide the output from `k3s -v` -->
+**Version Info:**
+<!-- Provide the output from "k3s -v" -->
+K3s Version: 
+<!-- Provide the output from "uname -a" on the node(s) -->
+Node(s) OS and Version: 
 
-**K3s arguments:**
-<!-- Provide the flags used to install or run k3s server. -->
-
-**Describe the bug**
+**Describe the bug:**
 <!-- A clear and concise description of what the bug is. -->
 
-**To Reproduce**
-<!-- Steps to reproduce the behavior: -->
+**Steps To Reproduce:**
+<!-- Steps to reproduce the behavior. Please include as the first step how you installed K3s on the node(s) (including all flags or environment variables) -->
```

Comment:
```suggestion
<!-- Steps to reproduce the behavior. Please include as the first step how you installed K3s on the node(s) (including all flags or environment variables). If you have customized configuration via [systemd drop-ins or overrides](https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html) please include those as well.-->
```

Reply:
the `[link](url)` will not render in `<!-- comments -->` we can change that or just leave it as it - I do not mind either way. @dweomer

Reply:
We could do something like this:

```
<!-- Steps to reproduce the behavior. Please include as the first step how you installed K3s on the node(s) (including all flags or environment variables). If you have customized configuration via systemd drop-ins or overrides (https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html) please include those as well.-->
```

Reply:
Addressed in https://github.com/rancher/k3s/pull/1767/commits/63f4e658454c57ed7da616d37efd20321c016cdb